tests/baseSetup.go

```diff
@@ -20,40 +20,44 @@ import (
 	"fmt"
```

Comment:
I think this file should move a level down. its only applicable for file, block, and and object storage suites which assume rook is already installed. it should not run, for example, when we are testing the operator itself.

maybe get rid of the this file here and just have each of the test suites that need rook installed call rookHelper directly?